### ISSUE TITLE
[PRE REVIEW]: PyTeNet: A concise Python implementation of quantum tensor network algorithms

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#946]
**Submitting author:** @cmendl (<a href="http://orcid.org/0000-0002-6386-0230">Christian B. Mendl</a>)
**Repository:** <a href="https://github.com/cmendl/pytenet" target ="_blank">https://github.com/cmendl/pytenet</a>
**Version:** v1.0
**Editor:** @jedbrown
**Reviewers:** mlxd

**Author instructions**

Thanks for submitting your paper to JOSS @cmendl. The JOSS editor (shown at the top of this issue) will work with you on this issue to find a reviewer for your submission before creating the main review issue.

@cmendl if you have any suggestions for potential reviewers then please mention them here in this thread. In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission.

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @jedbrown it looks like you're currently assigned as the editor for this paper :tada:

For a list of things I can do to help you, just type:

```
@whedon commands
```

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.00946/joss.00946/10.21105.joss.00946.pdf)

**Comment by arfon** [comment]
:wave: @jedbrown, the submitting author suggested you as the handling editor.

**Comment by cmendl** [comment]
Concerning the article proof: would it be possible to switch to numerical citation style? I've added a reference to american-physics-society.csl in paper.md but apparently this wasn't used for compiling the paper to pdf...

**Comment by jedbrown** [comment]
@cmendl The citation style is meant to be standard across JOSS publications. I hope that's not a problem for you.

@jcmgray This is in a similar space to your recent article #819. Would you be available to review or could you suggest a colleague?

@mlxd We'd be thrilled if you could review this time.

**Comment by mlxd** [comment]
Sure, I can review this.

**Comment by jcmgray** [comment]
I'd be more than happy to review, the only caveat being I am quite busy currently so it might be over the next few weeks.

**Comment by jedbrown** [comment]
@whedon add mlxd as reviewer

**Comment by whedon** [comment]
OK, mlxd is now a reviewer

**Comment by jedbrown** [comment]
@whedon add jcmgray is reviewer

**Comment by jedbrown** [comment]
Thanks to both of you.

**Comment by jedbrown** [comment]
@whedon start review

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/948. Feel free to close this issue now!

**Comment by jcmgray** [comment]
@jedbrown 
> add jcmgray **is** reviewer

I think this little typo might have missed me off as a reviewer.

**Comment by jedbrown** [comment]
@jcmgray Thanks for catching that. I fixed the review thread #948.
